### ISSUE TITLE
Bubble Bobble (Old Set) not loading (possibly broken?

### ISSUE THREAD

**Issue by KMFDManic** [libretro/mame2003-plus-libretro#283]
I tested bublbobl.zip on MAME 2003 Plus, and it failed to load.  It works fine with the current build of MAME 2003 from:

https://github.com/libretro/mame2003-libretro

**Comment by arcadez** [comment]
You checked your rom against the dat..?? reason i ask is you require a new rom
for use in this core.

**Comment by KMFDManic** [comment]
@arcadez Thanks for the heads-up on the new rom.  I just wanted clarification on this.

**Comment by arcadez** [comment]
@KMFDManic To get the sound to work in Outfoxies would require the M37710 CPU to be
backported to this core and hereby lies the problem, there was a huge memory rewrite
between MAME78.u2 and MAME79 this means that CPU is not compatable with this core

Trust me if it was possible i would have done it already as it's the same CPU needed to
get both the Namco Classic Collection games up and running with full sound and music
but alas it's not to be.

**Comment by KMFDManic** [comment]
@arcadez Obviously, you have checked into this in the past:)  Thank you for the clarification.
